modules/340-monitoring-kubernetes/hooks/helm.go

```diff
@@ -129,6 +135,13 @@ var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 func handleHelmReleases(input *go_hook.HookInput, dc dependency.Container) error {
 	input.MetricsCollector.Expire("helm_deprecated_apiversions")
 
+	var k8sConfigK8sVersion string
+	unsupportVersion.clean()
+	k8sConfigK8sVersionRaw, ok := input.Values.GetOk("global.clusterConfiguration.kubernetesVersion")
```

Comment:
It doesn't work.
```
$ deckhouse-controller global values -o json | jq '.clusterConfiguration.kubernetesVersion'
"1.27"
```

while
```
$ kubectl -n kube-system get secret d8-cluster-configuration -o jsonpath='{.data.cluster-configuration\.yaml}' | base64 -d  | grep "kubernetesVersion"
kubernetesVersion: Automatic
```

I have told that we have to add subscription to the secret and fetch kubernetes version from it

Reply:
That's where I started. But subscriptions don't work in "on startup" hooks.

Reply:
The functionality has been moved to a separate hook. The method of getting "kubernetesVersion" has been redone.